avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroSerializerConfig.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+package io.confluent.kafka.serializers;
+
+import io.confluent.common.config.ConfigDef;
+
+import java.util.Map;
+
+/**
+ * Created by Priti on 7/29/15.
+ */
+public class KafkaAvroSerializerConfig extends AbstractKafkaAvroSerDeConfig {
+
+    public static final String ENABLE_AUTO_SCHEMA_REGISTRATION_CONFIG = "enable.auto.schema.registration";
+    public static final boolean ENABLE_AUTO_SCHEMA_REGISTRATION_DEFAULT = true;
+    public static final String ENABLE_AUTO_SCHEMA_REGISTRATION_DOC =
+            "If true, allows schema creation";
```

Comment:
Can this doc string add more details? 

For example - if this is true, what is the behavior when serializing an object with a schema not in the registry? If false, what happens if the schema is not in the schema registry?